src/components/Login.js

```diff
@@ -1,14 +1,16 @@
-import React, { useState } from "react";
+import React from "react";
 import { Redirect } from "react-router-dom";
 import { useAuth } from "../context/auth";
 import axios from "axios";
 import { FE_HOMEPAGE_PARAM } from "../constant";
+import { useBoolean, useFormInput } from "../hooks";
+import Countdown from "./Countdown";
+import AnswerPanel from "./AnswerPanel";
 
 const Login = () => {
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-  const [loggedIn, setLoggedIn] = useState(false);
-  const [isError, setIsError] = useState(false);
+  const [values, handleChange] = useFormInput({ email: "", password: "" });
+  const { value: loggedIn, setTrue: setIsLoggedIn } = useBoolean(false);
```

Comment:
ini ga perlu se-arkward ini WWKWWK.
lgsg aja const email = useFormInput('')
const { loggedIn, setIsLoggedIn } = useBoolean(false)
feel free utk destructure or not, tapi pastinya ga harus "value" WWKWKWK